app/assets/stylesheets/base.css.erb

```diff
@@ -877,7 +877,6 @@ a.overlay-video
 		position:absolute;
 		background-repeat:no-repeat;
 		background-position:50% 50%;
-		#filter:alpha(opacity=0);
```

Comment:
Forgot to comment on these... I can only assume the "#" was meant as a comment event though it's not...

`sass-rails` appears to be a bit more strict on the compiled CSS as this resulting in a hard exception when it tried to build it. I removed the lines because, according to "Mozilla", they are used for IE8 and earlier compatibility and last I checked, it's 2020.